Utilities/rPlatform.cpp

```diff
@@ -54,7 +56,16 @@ std::string rPlatform::getConfigDir()
 		else // Just in case
 			dir = "./config";
 		dir = dir + "/rpcs3/";
-		mkdir(dir.c_str(), 0777);
+
+		s32 ret = mkdir(dir.c_str(), 0777);
```

Comment:
It may return -1, not an error code. Also use `std::printf` to report success/failure, LOG_ won't work correctly.